ci/tox/Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN set -x \
 ARG python_version
 ENV PYTHON_VERSION=${python_version}
 ENV PYTHON=python${python_version}
+RUN apk add --no-cache gcc musl-dev # both packages are required by tox
```

Comment:
It's required to install `mypy` right? not for `tox` itself to run (since `tox` is just a tool like `Make` that only installs&runs stuff but doesn't contain much logic on its own)

Reply:
I'd clarify that in the comment